### PR TITLE
New term PI4KA-Related Disorder

### LINKED ISSUE #7858: PI4KA-Related Disorder
PI4KA-Related Disorder

Synonyms: None

Parent Terms: MONDO:0030831, MONDO:0020340, MONDO:0015150

Definition:
Per GeneReviews PI4KA-related disorder is a clinically variable disorder characterized primarily by neurologic dysfunction (limb spasticity, developmental delay, intellectual disability, seizures, ataxia, nystagmus), gastrointestinal manifestations (multiple intestinal atresia, inflammatory bowel disease), and combined immunodeficiency (leukopenia, variable immunoglobulin defects). Age of onset is typically antenatal or in early childhood; individuals can present with any combination of these features. Rare individuals present with later-onset hereditary spastic paraplegia. Brain MRI findings can include hypomyelinating leukodystrophy, cerebellar hypoplasia/atrophy, thin or dysplastic corpus callosum, and/or perisylvian polymicrogyria.

Definition source: https://www.ncbi.nlm.nih.gov/books/NBK583038/

Children terms: MONDO:0014679, MONDO:0030482, MONDO:0030669

Working Group: https://www.clinicalgenome.org/affiliation/40020/

@ErinRiggs @elizabeth-gavrilov

### PR DESCRIPTION
**Pull request by yshwetar**
Closes #7858

- Definition
- Term tracker
- 3 Parent terms
- 3 Child terms
- PMID and Clingen links

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -574805,6 +574808,17 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-12T13:35:21Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7493" xsd:anyURI
 
+[Term]
+id: MONDO:1040012
+name: PI4KA-Related Disorder
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:574813]: "related" and "disorder" should not be capitalized

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -574805,6 +574808,17 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-12T13:35:21Z" xsd:
...
+def: "Characterized primarily by neurologic dysfunction (limb spasticity, developmental delay, intellectual disability, seizures, ataxia, nystagmus), gastrointestinal manifestations (multiple intestinal atresia, inflammatory bowel disease), and combined immunodeficiency (leukopenia, variable immunoglobulin defects). Age of onset is typically antenatal or in early childhood; individuals can present with any combination of these features. Rare individuals present with later-onset hereditary spastic paraplegia. Brain MRI findings can include hypomyelinating leukodystrophy, cerebellar hypoplasia/atrophy, thin or dysplastic corpus callosum, and/or perisylvian polymicrogyria." [https://www.clinicalgenome.org/affiliation/40020/, PMID:35951779]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:574814]: Definition should start with a "genus" (eg. "a disorder that is characterized....")
I think this term can also follow the "disease by gene" pattern.

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -574805,6 +574808,17 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-12T13:35:21Z" xsd:
...
+is_a: MONDO:0015150 {source="PMID:35951779", source="https://www.clinicalgenome.org/affiliation/40020/"} ! complex hereditary spastic paraplegia
+is_a: MONDO:0020340 {source="PMID:35951779", source="https://www.clinicalgenome.org/affiliation/40020/"} ! bilateral perisylvian polymicrogyria
+is_a: MONDO:0030831 {source="PMID:35951779", source="https://www.clinicalgenome.org/affiliation/40020/"} ! gastrointestinal defect and immunodeficiency syndrome
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:574817]: can we add the SC to the PI4KA gene?

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (yshwetar) [reply src/ontology/mondo-edit.obo:574813]: Comment addressed.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (yshwetar) [reply src/ontology/mondo-edit.obo:574814]: Comment addressed.

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (yshwetar) [reply src/ontology/mondo-edit.obo:574817]: Comment addressed.

**Review by sabrinatoro — APPROVED** [review]
Good job! 
Approving and merging.

**Pull request merged**